rules/builtins.build_defs

```diff
@@ -283,3 +283,7 @@ def is_semver(s:str) -> bool:
 
 def semver_check(version:str, constraint:str) -> bool:
     pass
+
+def base64_url(s:str) -> str:
```

Comment:
It'd be better to call this `base64url` (without the underscore), since that's the name of the function according to [RFC 4648](https://datatracker.ietf.org/doc/html/rfc4648#section-5), and the underscore implies that it only consumes URLs.